src/core/execute.c

```diff
@@ -846,6 +846,12 @@ static int setup_pam(
                         goto fail;
         }
 
+        STRV_FOREACH(e, *env) {
+                pam_code = pam_putenv(handle, *e);
+                if (pam_code != PAM_SUCCESS)
+                        goto fail;
```

Comment:
Is there any way to find out what `*e` (`var=value`) caused `pam_putenv` to fail?